tests/integration/test_workflow_documentation.py

```
@@ -40,304 +40,62 @@
 
-class TestDocumentationStructure:
-    """Test the structure and formatting of the documentation."""
-    
-    @pytest.fixture
-    def doc_content(self) -> str:
-        """Load the documentation content."""
+@pytest.fixture(scope='session')
+def doc_content() -> str:
+    """Load the documentation content once per test session."""
+    try:
         with open(DOC_FILE, 'r', encoding='utf-8') as f:
```

Ruff (0.14.6)

Unnecessary mode argument

Remove mode argument

(UP015)